### PR TITLE
Logging

### PR DESCRIPTION
**Pull request by MBishop17**
Hi Guys,

Can I please get a review on the draft I've written for logging?

### CODE REVIEW

File: content/rancher/v2.x/en/tools/logging/embedded-elasticsearch/_index.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
```
Comment (cloudnautique) [inline content/rancher/v2.x/en/tools/logging/embedded-elasticsearch/_index.md]: This section needs to be removed.

File: content/rancher/v2.x/en/tools/logging/_index.md [hunk 2]
```diff
@@ -5,6 +5,15 @@ aliases:
   - /rancher/v2.x/en/tasks/logging/
 ---
 
-Rancher has the capability to push out cluster and project logs to an external
-log monitoring tool such as Splunk or a Syslog server. This allows you to be
-alerted of errors and warnings in your Kubernetes infrastructure.
+To record cluster or project events, Rancher can integrate with a variety of popular logging services that exist outside of your clusters. Configuring logging services:
```
Comment (cloudnautique) [inline content/rancher/v2.x/en/tools/logging/_index.md]: not 'events' these are logs.

Comment (cloudnautique) [reply content/rancher/v2.x/en/tools/logging/_index.md]: I would also say popular logging services and tools.

File: content/rancher/v2.x/en/tools/logging/_index.md [hunk 3]
```diff
@@ -5,6 +5,15 @@ aliases:
...
+
+- Alerts you to errors and warnings in your Kubernetes infrastructure in a live stream.
```
Comment (cloudnautique) [inline content/rancher/v2.x/en/tools/logging/_index.md]: Allows you to caputure and analyze the state of your cluster and look for trends in your environment from the log stream. Live stream, might have some delay, so I would say stream.

File: content/rancher/v2.x/en/tools/logging/_index.md [hunk 4]
```diff
@@ -5,6 +5,42 @@ aliases:
...
+- [Elasticsearch]({{< baseurl >}}/rancher/v2.x/en/tools/logging/elasticsearch)
+- [Splunk]({{< baseurl >}}/rancher/v2.x/en/tools/logging/splunk)
+- [Kafka]({{< baseurl >}}/rancher/v2.x/en/tools/logging/kafka)
+- [Syslog]({{< baseurl >}}/rancher/v2.x/en/tools/logging/syslog) 
+
+## Advantages
+
+Setting up a logging service to collect logs from your cluster/project is helpful several ways:
+
+- Alerts you to errors and warnings in your Kubernetes infrastructure in a stream. The stream informs you of events like a container crashing, a pod eviction, or a node dying.
```
Comment (superseb) [inline content/rancher/v2.x/en/tools/logging/_index.md]: It's only logging, not alerting. The second line is more accurate (informs)

File: content/rancher/v2.x/en/tools/logging/_index.md [hunk 5]
```diff
@@ -5,15 +5,42 @@ aliases:
...
+## Advantages
+
+Setting up a logging service to collect logs from your cluster/project is helpful several ways:
+
+- Logs errors and warnings in your Kubernetes infrastructure to a stream. The stream informs you of events like a container crashing, a pod eviction, or a node dying.
+- Allows you to capture and analyze the state of your cluster and look for trends in your environment using the log stream.
+- Helps you when troubleshooting or debugging.
+- Saves your logs to a safe location outside of your cluster, so that you can still access them even if your cluster encounters issues.
+
+## Logging Scope
```
Comment (MBishop17) [inline content/rancher/v2.x/en/tools/logging/_index.md]: @cloudnautique , how does the scope of logging change for clusters vs. project? What additional logs do you get at the cluster level?

Comment (cloudnautique) [reply content/rancher/v2.x/en/tools/logging/_index.md]: Cluster level in 2.0.7 logs every pod in the cluster's logs including the Kubernetes system components from an RKE cluster.

In a coming release, you will be able to send the kubernetes system components(Kubelet, kube-apiserver, etc.) to their own index for an operator to monitor and only user workloads would be sent at the cluster level. There will likely be an option to include system level at the cluster level.

The main goal is that there are 3 audiences potentially:
Cluster operator - System project logs
Cluster Operator/Audit/Security team - Cluster logs
Project Logs - Developer/end user scop

Comment (MBishop17) [reply content/rancher/v2.x/en/tools/logging/_index.md]: @cloudnautique , I made one more update based on what you said here. Can I get one more review?

**Comment by cloudnautique** [comment]
LGTM

**Pull request merged**